packages/features/ee/workflows/components/EventWorkflowsTab.tsx

```diff
@@ -223,13 +232,17 @@ function EventWorkflowsTab(props: Props) {
       });
       const disabledWorkflows = data.workflows.filter(
         (workflow) =>
+          (!workflow.teamId || eventType.teamId === workflow.teamId) &&
           !workflows
             .map((workflow) => {
               return workflow.id;
             })
             .includes(workflow.id)
       );
-      setSortedWorkflows(activeWorkflows.concat(disabledWorkflows));
+      const allSortedWorkflows = workflowsDisableProps.isLocked
+        ? activeWorkflows
+        : activeWorkflows.concat(disabledWorkflows);
+      setSortedWorkflows(allSortedWorkflows);
```

Comment:
If it is locked we only want to show active workflows. Which should only be active team workflows and no personal workflows